AFQ/api/bundle_dict.py

```diff
@@ -1021,6 +976,12 @@ def is_bundle_in_template(self, bundle_name):
         return "space" not in self._dict[bundle_name]\
             or self._dict[bundle_name]["space"] == "template"
 
+    def _roi_distance_helper(self, roi_or_sl):
+        return nib.Nifti1Image(
+            distance_transform_edt(
+                np.where(roi_or_sl.get_fdata() == 0, 1, 0)),
```

Comment:
Just making sure that I am understanding this correctly: here the "target" (i.e, the ROI  or sl) is considered "background" (as described in the distance_transform_edt dostring), because you are computing the distance to the target (and that function computes distance to the "background"). Is that correct?

Reply:
Yes! As you point out, it's due to scipy's api for distance_transform_edt